src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -232,7 +232,8 @@
     <xsd:complexType name="workflow">
         <xsd:sequence>
             <xsd:element name="marking-store" type="marking_store" />
-            <xsd:element name="support" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="support" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="support-strategy" type="xsd:string" minOccurs="0" maxOccurs="1" />
```

Comment:
`minOccurs="0" maxOccurs="1"` it's already bounded. so it's not possible to have many element.
I did not set an attribute to be consistent with `support`.